src/certificates/layout/index.jsx

```diff
@@ -0,0 +1,71 @@
+import PropTypes from 'prop-types';
+import {
+  Container, Button, Layout, Dropdown, DropdownButton,
+} from '@edx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
+
+import SubHeader from '../../generic/sub-header/SubHeader';
+import CertificatesSidebar from '../certificates-sidebar';
+import messages from '../messages';
+
+const MainLayout = ({ courseId, showHeaderButtons, children }) => {
+  const intl = useIntl();
+  return (
+    <Container size="xl" className="certificates px-4">
+      <div className="mt-5" />
+      <SubHeader
+        hideBorder
+        subtitle={intl.formatMessage(messages.headingSubtitle)}
+        title={intl.formatMessage(messages.headingTitle)}
+        headerActions={showHeaderButtons && ( // TODO create btn handlers in the task (https://youtrack.raccoongang.com/issue/AXIMST-166)
+          <>
+            <DropdownButton id="dropdown-basic-button">
+              <Dropdown.Item />
+            </DropdownButton>
+            <Button
+              variant="outline-primary"
+              alt={intl.formatMessage(messages.headingActionsPreviewAlt)}
```

Comment:
[important] Since we are displaying a decorative image inside a button, we do not need to specify alt for it. In our case, the screen reader will tell the user that this is a button and its text.